docs/AccessControl.md

```diff
@@ -34,89 +35,93 @@ If a correct passphrase is not provided by the client (caller), the request from
 **Note!** `srt_listen_callback(...)` can't check the passphrase directly for security reasons. 
 The only way to make the app check the passphrase is to set the passphrase on the socket by using the `SRTO_PASSPHRASE` option. This lets SRT to reject connection on mismatch.
 
-
 ## Character Encoding
 
 The Stream ID uses UTF-8 encoding.
 
 ## General Syntax
 
 This recommended syntax starts with the characters known as an executable
-specification in POSIX: 
-
-`#!` 
+specification in POSIX: `#!`.
 
 The next two characters are:
 
 - `:` - this marks the YAML format, the only one currently used
 - The content format, which is either:
-   - `:` - the comma-separated keys with no nesting
-   - `{` - like above, but nesting is allowed and must end with `}`
+  - `:` - the comma-separated keys with no nesting
+  - `{` - like above, but nesting is allowed and must end with `}`
 
 (Nesting means that you can have multiple level brace-enclosed parts inside.)
 
 The form of the key-value pair is:
 
-- `key1=value1,key2=value2`...
+```js
+key1=value1,key2=value2...
+```
 
 ## Standard Keys
 
-Beside the general syntax, there are several top-level keys treated as standard 
-keys. Other keys can be used as needed.
+Beside the general syntax, there are several top-level keys treated as standard
+keys.
```

Comment:
This text would be rich-formatted as a continuous text. Use either explicit extra line for paragraph split, or join whole text into one paragraph, was that intended.

Reply:
Fixed (as one paragraph).